### PR TITLE
poster of video 

### PR DESCRIPTION
**Pull request by choukin**
## Solve the poster does not echo

### CODE REVIEW

File: src/trait_manager/model/Trait.js [hunk 1]
```diff
@@ -67,6 +67,11 @@ export default Backbone.Model.extend({
 
     if (this.get('changeProp')) {
       target.set(name, value, opts);
+      if (name === 'poster') {
```
Comment (artf) [inline src/trait_manager/model/Trait.js]: You're putting some specific condition (the poster attribute from video) in a generic object (Trait) and this is not good from the architectural and logical point of view. At least, you should put that condition in the Video Component

File: index.html [hunk 2]
```diff
@@ -121,6 +122,11 @@ <h1 class="welcome">Welcome to</h1>
         attributes: { class:'gjs-fonts gjs-f-b1' },
         content: `<div style="padding-top:50px; padding-bottom:50px; text-align:center">Test block</div>`
       })
+      editor.BlockManager.add('Video', {
+        label: 'Video',
+        attributes: { class:'gjs-fonts gjs-f-b1' },
+        content: `<video  />`
+      })
     </script>
```
Comment (artf) [inline index.html]: Leave index.html as it is

File: src/dom_components/model/ComponentVideo.js [hunk 3]
```diff
@@ -182,8 +182,8 @@ export default Component.extend(
         {
           label: 'Poster',
           name: 'poster',
-          placeholder: 'eg. ./media/image.jpg',
-          changeProp: 1
+          placeholder: 'eg. ./media/image.jpg'
+          // changeProp: 1
```
Comment (artf) [inline src/dom_components/model/ComponentVideo.js]: So the PR is just about changeProp in poster trait, right?

Comment (choukin) [reply src/dom_components/model/ComponentVideo.js]: yes

Comment (choukin) [reply src/dom_components/model/ComponentVideo.js]: > So the PR is just about changeProp in poster trait, right?

yes

File: src/commands/view/OpenTraitManager.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ export default {
         .trigger('change:appendContent');
 
       this.target = editor.getModel();
+      debugger;
```
Comment (artf) [inline src/commands/view/OpenTraitManager.js]: remove the debugger

File: index.html [hunk 5]
```diff
@@ -16,6 +16,7 @@
 
   <body>
     <div id="gjs" style="height:0px; overflow:hidden;">
+      <video  src="" poster="https://camo.githubusercontent.com/cf1a042fe34ab268410fdf6050699f20cf0aaef2/68747470733a2f2f796d6d2d6d616c69616e672e6f73732d636e2d68616e677a686f752e616c6979756e63732e636f6d2f796d6d2d6d616c69616e672f6163636573732f796d6d5f313533393538383635353835302e706e67"></video>
```
Comment (artf) [inline index.html]: remove the video, please

**Comment by artf** [comment]
👍

**Pull request merged**